subsys/bluetooth/audio/ascs.c

```diff
@@ -1446,6 +1508,33 @@ static ssize_t ascs_config(struct bt_ascs *ascs, struct net_buf_simple *buf)
 	return buf->size;
 }
 
+int bt_audio_ep_get_info(const struct bt_audio_ep *ep, struct bt_audio_ep_info *info)
+{
+	info->id = ep->status.id;
+	info->state = ep->status.state;
+	info->dir = ep->dir;
+
+	return 0;
+}
```

Comment:
This should be put in `stream.c` I think, so that it can be used as the unicast client as well